panel_view.py

```diff
@@ -92,12 +107,15 @@ def on_post_save_async(self, view):
         if show_panel_on_save == 'never' or panel_is_active(window):
             return
 
-        errors_by_bid = get_window_errors(window, persist.errors)
+        bid = view.buffer_id()
+        State['awaited_views'].add(bid)
 
-        if show_panel_on_save == 'window' and errors_by_bid:
-            window.run_command("show_panel", {"panel": OUTPUT_PANEL})
-        elif view.buffer_id() in errors_by_bid:
-            window.run_command("show_panel", {"panel": OUTPUT_PANEL})
+        if buffers_effective_lint_mode_is_background(bid):
+            show_panel_if_errors(window, bid)
+
+    def on_modified_async(self, view):
+        bid = view.buffer_id()
+        State['awaited_views'].discard(bid)
```

Comment:
Perhaps we need to do the same thing `on_close(view)`?

Reply:
Yes

Reply:
Also awaited_view -> just_saved_buffers. The important part here is to have buffer in the name bc that's what's usually confusing.